From: Johannes Thumshirn <jthumshirn@suse.de>
Date: Tue 22 May 11:58:05 CEST 2018
Subject: kabi protect struct acpi_nfit_desc
References: bsc#1091424
Patch-mainline: Never, KABI fixup

Move additional new structure members to the back of acpi_nfit_desc.

Signed-off-by: Johannes Thumshirn <jthumshirn@suse.de>
---
 drivers/acpi/nfit/nfit.h |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- a/drivers/acpi/nfit/nfit.h
+++ b/drivers/acpi/nfit/nfit.h
@@ -196,7 +196,6 @@ struct acpi_nfit_desc {
 	struct device *dev;
 	u8 ars_start_flags;
 	struct nd_cmd_ars_status *ars_status;
-	struct nfit_spa *scrub_spa;
 	struct delayed_work dwork;
 	struct list_head list;
 	struct kernfs_node *scrub_count_state;
@@ -212,6 +211,9 @@ struct acpi_nfit_desc {
 	unsigned int scrub_tmo;
 	int (*blk_do_io)(struct nd_blk_region *ndbr, resource_size_t dpa,
 			void *iobuf, u64 len, int rw);
+#ifndef __GENKSYMS__
+	struct nfit_spa *scrub_spa;
+#endif
 };
 
 enum scrub_mode {
